wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -613,19 +613,23 @@ There are two types of image links: inline and reference. Examples:
 **Inline style:**
 
 ```markdown
-![](/wiki/shared/flag/AU.gif)
+![Australian flag](/wiki/shared/flag/AU.gif "Australia")
```

Comment:
flags are kinda bad examples at this point because they're supporting icons that don't need alt text.

I think a good example would be the placement crowns from tournaments, since they showcase the difference between alt & title text

Reply:
+1